### ISSUE TITLE
cy.exec sources profile once when opening from Finder / Windows Explorer

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#694]
- Operating System: Mac
- Cypress Version: 0.20.1
- Browser Version: Electron

### Is this a Feature or Bug?

A 🐞

### Current behavior:

A test that executes two `node` commands fails on the second command IF we start Cypress by double clicking the app in the Finder program

```js
cy.exec('node --version')
cy.exec('node --version')
```

![screen shot 2017-09-28 at 4 35 23 pm](https://user-images.githubusercontent.com/2212006/30989521-02080e70-a46c-11e7-8093-27c6c3b8ce0f.png)

![screen shot 2017-09-28 at 4 37 14 pm](https://user-images.githubusercontent.com/2212006/30989529-072ce344-a46c-11e7-8e80-f45fc9157ce5.png)

Note the first command executes correctly! But the second has "node unknown command"

### Desired behavior:

If we start Cypress "normally" from the shell, then it works fine (because it inherits shell functions, like NVM node setup).

![screen shot 2017-09-28 at 4 33 16 pm](https://user-images.githubusercontent.com/2212006/30989586-2bbac442-a46c-11e7-803c-eac01edc4ff4.png)

### How to reproduce:

I have a repo https://github.com/bahmutov/foo-page with just this test

#### Test code:

```js
it('works', () => {
  cy.exec('node --version')
  cy.exec('node --version')
})
```

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.2`](https://docs.cypress.io/guides/references/changelog.html#0-20-2).
